documentation20/en/08.connector/01.java/docs.md

```diff
@@ -148,6 +149,16 @@ The JDBC-RESTful does not depend on the local function library. Compared with JD
 * JdbcUrl starts with "JDBC:TAOS-RS://"
 * Use port 6041 as the connection port
 
+Starting with version taos-jdbcdriver-2.0.38 and TDengine 2.4.0.12, **JDBC-RESTful** support batch load function. Data is transmitted between TAOS-JDBcDriver and TDengine through WebSocket connection. Compared with HTTP, WebSocket enables **JDBC-restful** to support large data volume query and improves query performance.
```

Comment:
official feature name is bulk pulling

Reply:
My version:
JDBC 2.0.38 and later version with TDengine 2.4.0.12 (and later version) start to support a new bulk-pulling feature. It provides higher data transmission performance and larger volume data queries capability via WebSocket communication.